src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
```

Comment:
```suggestion
  constructor(name) {
  this.name = name;
  this.health = 100
```

Think you dont need this health value as parameter bcs every animal will start with 100 health by default

Reply:
I set health to 100 in body.